src/api/verifyEmail.js

```diff
@@ -0,0 +1,45 @@
+const router = require("express").Router();
+const User = require("../models/User");
+const { sendResponse } = require("./../utils/sendResponse");
+const { isGmailEnabled } = require("../utils/getConfigFile");
+const { sendMail } = require("../utils/sendMail");
+
+router.post("/verifyEmail", async function(req, res) {
+  const usingGmail = await isGmailEnabled();
+  if (!usingGmail) {
+    return sendResponse(
+      res,
+      500,
+      "Gmail not enabled. Do not use this endpoint."
+    );
+  }
+  if (!req.body || !req.body.email || !req.body.pin) {
+    return sendResponse(
+      res,
+      400,
+      "Malformed request: email or pin not specified"
+    );
+  }
+
+  const user = await User.findOne({ email: req.body.email }).catch(e =>
+    console.log(e)
```

Comment:
Let's send the error as a part of te response eg 
```
 return sendResponse(res, 400, e);
```

Reply:
Great idea!